kanidm_book/src/integrations/oauth2.md

```diff
@@ -270,4 +270,35 @@ these to a group with a scope map due to Velociraptors high impact.
 
     # kanidm group create velociraptor_users
     # kanidm group add_members velociraptor_users ...
-    kanidm system oauth2 create_scope_map <resource server name> velociraptor_users openid email
\ No newline at end of file
+    kanidm system oauth2 create_scope_map <resource server name> velociraptor_users openid email
+
+### Vouch Proxy
+
+_You need to run at least the version 0.37.0_.
+
+Vouch Proxy supports multiple OAuth and OIDC login providers.
+To configure it you need to pass:
+
+```yaml
+oauth:
+  auth_url: https://idm.wherekanidmruns.com/ui/oauth2
+  callback_url: https://login.wherevouchproxyruns.com/auth
+  client_id: <oauth2_rs_name> # Found in kanidm system oauth2 get XXXX (should be the same as XXXX)
+  client_secret: <oauth2_rs_basic_secret> # Found in kanidm system oauth2 get XXXX
+  code_challenge_method: S256
+  provider: oidc
+  scopes:
+    - login
+    - openid
+    - read_user
+    - profile
+    - email # <<<<< Important
```

Comment:
Could it be worth explaining why email is important, and what these other scopes are for?

Reply:
Done, the other scopes are not, it was leftovers of me playing around, sorry about that